R/misc.R

```diff
@@ -107,7 +107,6 @@ factor_check <- function(base, rec, add, colors = TRUE) {
     purrr::map_lgl(rec$template %>% dplyr::select(dplyr::one_of(nominal)),
                    rlang::is_character)
   if (any(is_str)) {
-    nominal <- rlang::syms(nominal[is_str])
```

Comment:
Do you still need to do the `is_str` subset? so `nominal <- nominal[is_str]`

Reply:
Oh gosh 🙈 yep